_posts/2020-12-11-an-old-new-adventure.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title:  "An Old New Adventure"
+date:   2020-12-16 09:28:42 -0700
+categories: [personal]
+---
+After two and a half years of being independent, I am returning to Google.
+
+The time off was really valuable. I was still feeling residual effects from burnout on the Android team in late 2015, and also drained by family and personal things that were happening and needed more time and energy. I got that, and return recharged and with some insights that I hope will be useful. I'll touch on a few of those in this post. Each could probably be its own blog post, but today I want to briefly note the event.
+
+## On open source sustainability
+
+Much has been written on open source sustainability, notably [Nadia Eghbal]'s *Working in Public.* I won't speak to open source more broadly (except to note how impressed I am with Blender and Krita), but for the specific task of building an ecosystem for a library, I think there is one model that actually works: being hired by a company that depends on that ecosystem.
+
+To some extent, that's an indictment of our capitalist system. In an ideal universe, there would be strong institutions dedicated to the public interest where open source developers could develop, researchers could research, and spend an absolute minimum of time and energy hustling for support. For software, in any case, universities are [not that,][stein on leaving ucsd] otherwise I'd be quite tempted. In the actual world, working for a company like Google is about as close as you can come.
+
+I remain skeptical of patronage-style platforms such as Patreon or Github Sponsors. I think it's possible to make them work, but only for a small number of fortunate people, and even then, the incentives for creating maximum value aren't that well aligned with the incentive structure of hustling on social media.
+
+So me (re-)joining Google full time is basically a statement of confidence in the model of being employed to work on open source.
```

Comment:
I get a very faint hint of “this is *the* successful open source funding model” here—or perhaps more accurately I should say that I think some people will read it this way. Might be worth hedging the wording a little, that there may be or are other successful models? Or not, as you like.

Reply:
Good point, I'll reword a bit.